PrimeWhitespace/solution_1/Dockerfile

```
@@ -0,0 +1,25 @@
+FROM ubuntu:24.04
+
+WORKDIR /tmp
+RUN apt-get update && \
+    apt-get install -y ghc git make python3 && \
+    git clone --depth 1 https://github.com/TryItOnline/WSpace && \
+    apt-get remove -y git && \
+    apt-get autoremove -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /tmp/WSpace
+RUN apt-get update && \
+    apt-get install -y make && \
+    make && \
+    cp wspace /usr/bin/whitespace && \
+    apt-get remove -y make && \
+    apt-get autoremove -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/WSpace
```

I think this can be combined/compacted as follows:

```suggestion
RUN apt-get update && \
    apt-get install -y ghc git make python3 && \
    git clone --depth 1 https://github.com/TryItOnline/WSpace && \
    make -C WSpace && \
    cp WSpace/wspace /usr/bin/whitespace && \
    apt-get remove -y git make && \
    apt-get autoremove -y && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/* /tmp/WSpace
```

---

I forgot about `make -C`. The only reason why I did it the way I did was due to a `hadolint` warning about using `WORKDIR` instead of the `cd`s that I was doing. This looks closer to what I originally had